code/dataobjects/QueuedJobRule.php

```diff
@@ -0,0 +1,159 @@
+<?php
+
+use AsyncPHP\Doorman\Rule;
+
+/**
+ * @property Int $Processes
+ * @property Varchar $Handler
+ * @property Decimal $MinimumProcessorUsage
+ * @property Decimal $MaximumProcessorUsage
+ * @property Decimal $MinimumMemoryUsage
+ * @property Decimal $MaximumMemoryUsage
+ * @property Decimal $MinimumSiblingProcessorUsage
+ * @property Decimal $MaximumSiblingProcessorUsage
+ * @property Decimal $MinimumSiblingMemoryUsage
```

Comment:
These properties actually return basic php types. I don't think you can get the DB field type unless you call `$this->obj('MinimumSiblingMemoryUsage')`